sqldelight-gradle-plugin/src/test/kotlin/com/squareup/sqldelight/SqlDelightPluginTest.kt

```diff
@@ -140,6 +140,19 @@ class SqlDelightPluginTest {
             + "7\t\tFROM test");
   }
 
+  @FixtureName("forbidden-column-name")
+  @Test
+  fun forbiddenColumnName() {
+    val result = fixture.executeAndFail()
+
+    assertThat(result.standardError).contains(
+        "Table.sq line 2:2 - Column name 'table_name' forbidden\n"
+            + "1\t\tCREATE TABLE test (\n"
+            + "2\t\t  table_name STRING NOT NULL\n"
+            + " \t\t  ^^^^^^^^^^^^^^^^^^^^^^^\n"
```

Comment:
oh...oh interesting. It could still have to do with spaces since it gets parsed down to

```
RULE: column definition
--RULE: column identifier
----LEAF: 'table_name'
--RULE: column_type
----LEAF: 'STRING'
--RULE: column_constraint
----LEAF: 'NOT NULL'
```

it might be ignoring spaces between the rules.